build.gradle

```diff
@@ -2106,6 +2106,8 @@ project(':tools') {
     implementation project(':server-common')
     implementation project(':connect:runtime')
     implementation project(':tools:tools-api')
+    implementation project(':transaction-coordinator')
+    implementation project(':group-coordinator')
```

Comment:
this is unnecessary, right?

Reply:
we need `:transaction-coordinator'`, but don't need `:group-coordinator'` in this PR, I will delete it